test/test_db.py

```diff
@@ -149,17 +149,23 @@ def test_write_document_index(document_store):
 
 
 @pytest.mark.elasticsearch
-@pytest.mark.parametrize("document_store", ["elasticsearch"], indirect=True)
-def test_write_document_with_embeddings(document_store):
+@pytest.mark.parametrize("document_store", ["elasticsearch", "faiss", "memory"], indirect=True)
+def test_document_with_embeddings(document_store):
     documents = [
         {"text": "text1", "id": "1", "embedding": np.random.rand(768).astype(np.float32)},
         {"text": "text2", "id": "2", "embedding": np.random.rand(768).astype(np.float64)},
-        {"text": "text3", "id": "3", "embedding": np.random.rand(768).astype(np.float32).tolist()},
-        {"text": "text4", "id": "4", "embedding": None},
+        {"text": "text3", "id": "3", "embedding": np.random.rand(768).astype(np.float32)},
```

Comment:
Why getting rid of these two cases? Are they not supported anymore?

Reply:
In the `FAISSDocumentStore`, the vectors get indexed in a batch. Currently, either all documents should have embeddings or none of them should have. Indexing a list of documents where only some have embeddings is not supported.